test/test_broker/acceptance/service-fabrik-api-2.0.instances.director.spec.js

```diff
@@ -2152,7 +2009,7 @@ describe('service-fabrik-api-sf2.0', function () {
             space_guid: space_guid,
             service_plan_guid: plan_guid
           });
```

Comment:
Same here, ```getServiceInstance``` will not be called for k8s case, maybe you can add that test.

Reply:
Check for all uses of ```getServiceInstance``` in this file and see if k8s test case can be added.

Reply:
Same comment for file: service-fabrik-api.instances.director.spec.js

Reply:
Currently end-to-end flow including auth for k8s is unclear, hence added single UT to cover the k8s branch as of now.